src/index/coinstatsindex.cpp

```diff
@@ -135,7 +157,17 @@ bool CoinStatsIndex::WriteBlock(const CBlock& block, const CBlockIndex* pindex)
                 Coin coin = Coin(out, pindex->nHeight, tx->IsCoinBase());
 
                 // Skip unspendable coins
-                if (coin.out.scriptPubKey.IsUnspendable()) continue;
+                if (coin.out.scriptPubKey.IsUnspendable()) {
```

Comment:
fc621a8fbdb5ecad1866e754ad38c24bfedfe5c1  This can in theory also happen for other (future) reasons than `OP_RETURN`:
```cpp
    /**
     * Returns whether the script is guaranteed to fail at execution,
     * regardless of the initial stack. This allows outputs to be pruned
     * instantly when entering the UTXO set.
     */
    bool IsUnspendable() const
    {
        return (size() > 0 && *begin() == OP_RETURN) || (size() > MAX_SCRIPT_SIZE);
    }
```

Reply:
Yepp, to be honest I just didn't come up with a better name in the moment and didn't revisit it afterwards. Will give this some more thought of what a good name is. Maybe `unspendable_script`.

Reply:
Renamed